heimdall-middleware-spec/src/test/java/br/com/conductor/heimdall/middleware/util/helpermock/call/RequestMockTest.java

```diff
@@ -0,0 +1,86 @@
+package br.com.conductor.heimdall.middleware.util.helpermock.call;
+
+import br.com.conductor.heimdall.middleware.spec.Header;
+import br.com.conductor.heimdall.middleware.spec.Query;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+
+import java.util.HashMap;
+
+public class RequestMockTest {
+
```

Comment:
You could import a static assert and the test class to reduce the length of the line. They try to keep the length less than 80 characters
`import static org.junit.Assert.*;`

so your line would be `assertEquals()` rather than `Assert.assertEquals()`